Gemfile

```diff
@@ -13,12 +13,12 @@ gem "prometheus_exporter", "~> 0.4.5"
 gem "rake"
 
 gem "kubeclient", :git => "https://github.com/abonas/kubeclient", :branch => "master"
-gem "manageiq-loggers", "~> 0.4.0", ">= 0.4.2"
+gem "manageiq-loggers", "~> 0.5.0"
```

Comment:
@slemrmartin I kind of recall you talking about this somewhere else, but since I'm not sure where I wanted to ask if this should move over to `providers-common` so each repo does not have to specify it.

Reply:
@gmcculloug 
I agree these common gems can be moved to providers-common

But there is a question if it's ok for all the common gems, maybe sometimes it's better to only specify minimal version in the providers-common.

Definitely it'll be good to talk and specify the policy about versioning in the near future

Reply:
We should probably discuss a plan for how to deal with versioning. I'm concerned about the potential confusion if we don't excecute consistently.